app/Http/Controllers/PhotoController.php

```diff
@@ -201,6 +213,10 @@ public function setDescription(Request $request)
 	}
 
 	/**
+	 * Define if a photo is public.
+	 * We do not advise the use of this and would rather see people use albums visibility
+	 * This would highly simplify the code if we remove this. Do we really want to keep it ?
+	 *
```

Comment:
One case where you can't use album visibility is for Unsorted. But the whole Unsorted always struck me as a bit of a strange feature. If we want to support pictures being displayed directly at the top level (without putting them in albums) then that's fine, but that's not what Unsorted does. When you upload a picture at the top level (albums view) it ends up in Unsorted. When you make it public, it also shows up in the Public smart album, but it's still not visible at the top level. What's the use case here? No idea...

Reply:
use case is being able to share a single picture in a private album.
You can then access the picture with the view.php (in Lychee v3) and a similar system in the v4.

Reply:
Yeah, I know. I guess what I was trying to say is that I would find the existing use case a lot more compelling if it applied to top-level images as well -- but there's currently no such thing as top-level images...